### PR TITLE
CA-296532: mpathalert log spam

### PR DESCRIPTION
**Pull request by minli1**
Disable debug log level for brand 'mscgen'

Signed-off-by: Min Li <min.li1@citrix.com>

### CODE REVIEW

File: http-svr/xmlrpc_client.ml [hunk 1]
```diff
@@ -371,6 +373,7 @@ module Protocol = functor(F: FORMAT) -> struct
     (* Caution: req can contain sensitive information such as passwords in its parameters,
        		 * so we should not log the parameters or a string representation of the whole thing.
        		 * The name should be safe though, e.g. req.Rpc.name when F is XMLRPC. *)
+    initialize_logging;
```
[the remark this answers is not in the document]
Comment (minli1) [reply http-svr/xmlrpc_client.ml]: Updated, thanks.

File: http-svr/xmlrpc_client.ml [hunk 2]
```diff
@@ -21,6 +21,8 @@ open D
 
 module E = Debug.Make(struct let name = "mscgen" end)
 
+let initialize_logging () = Debug.disable ~level:Syslog.Debug "mscgen"
```
Comment (gaborigloi) [inline http-svr/xmlrpc_client.ml]: Does it work if we do it once here:
```ocaml
let () = Debug.disable ~level:Syslog.Debug "mscgen"
```
That would be more efficient than calling this in every rpc call.

Comment (minli1) [reply http-svr/xmlrpc_client.ml]: Yes, good point. I will change and test. thanks.

**Comment by minli1** [comment]
Test passed, help to merge please. Thanks.

**Pull request merged**